scripts/run-pre-build-checks

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+# `$-` yields the list of the options of the current shell; we'll only interested in whether `x` is present
+x_option=${-//[^x]/}
```

Comment:
`$-`, but with all occurences (`//`) of regex `[^x]` replaced (`/`) with an empty string (nothing after the final `/`)